scalableminds/webknossos
# Proofreading without skeletons

Allow to proofread by clicking on segments instead of agglomerate skeleton nodes.
The position of the active segment is now updated when clicking onto a segment and also highlighted in proofreading mode using a small white cross hair. Afterwards, users can right-click somewhere else and select the entry to split or merge (a) segment(s).
Additionally, the active segment is highlighted a bit more than the currently hovered segment to make it more prominent and more easily recognizable.

### URL of deployed dev instance (used for testing):
- https://proofreadingwithoutskeletons.webknossos.xyz

### Steps to test:
- Create an annotation, activate an agglomerate view mapping and select the proofreading tool
  - Click on some segments - the click position should be highlighted with a zoom-independent cross hair + the active segment should be highlighted with higher opacity
  - Right-click somewhere else and select "Merge with active Segment" or "Split from active Segment"
- Open datasets with different scales - the cross hair should always have the same size.
- Use "Shift + Leftclick" and "Ctrl + Leftclick" to merge/split segments without the context menu. The description in the bottom bar should adapt automatically.
- Loading agglomerate skeletons and deleting/adding edges should still work for proofreading, although it's less discoverable (by design, might be removed in the near future)

### Issues:
- fixes #6507

------
(Please delete unneeded items, merge only when none are left open)
